runtime/arctic/src/lib.rs

```diff
@@ -435,6 +434,8 @@ parameter_types! {
 	pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 1_000_000_000u128);
 }
 
+pub type WeightToFee = constants::fee::WeightToFee;
```

Comment:
Any reason we can't assign this directly rather than wrapping it up in another type?